packages/react/src/hooks/index.ts

```diff
@@ -1 +1,3 @@
-export * from './useDataRef';
+export * from './useNotifications';
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** btw I'm missing the `package.json` export changes for the hooks, please check how we do this in the js package and do it similarly

Reply:
@LetItRock is this comment still pending or can we resolve it?

Reply:
@SokratisVidros we can resolve it, the exports were added